tests/FakeItEasy.IntegrationTests/TypeCatalogueTests.cs

```diff
@@ -150,7 +150,7 @@ private static string CaptureConsoleOutput(Action action)
 
         private string GetPathToOriginalExternalDll()
         {
-            var executingAssemblyPath = new Uri(this.GetType().GetTypeInfo().Assembly.Location).LocalPath
+            var executingAssemblyPath = new Uri(this.GetType().GetTypeInfo().Assembly.CodeBase).LocalPath

```

Comment:
I had been wondering about line 161 (as I'm sure you have already). At first I thought it weird to still be `.Location`, but I it's the copied that fails to load. Okay. I got my head around it. Thanks!